packages/frontend/src/services/indexer/cached-api.js

```diff
@@ -21,6 +22,7 @@ export default {
             kind: LIKELY_TOKENS_KEY,
             updater: (timestamp) => api.listLikelyTokens(accountId, timestamp),
             timeoutNs: UPDATE_REQUEST_INTERVAL_NS,
+            resetMs: RESET_REQUEST_INTERVAL_MS,
```

Comment:
Please add the same argument to `listLikelyNfts()` above; we can assume that since both caches use the same infrastructure/logic, either one could be inaccurate and need resetting.